### DISCUSSION TITLE
Is it possible to use koboDrmRemover.py in terminal, and pass my UserKey and DeviceID ?

### DISCUSSION THREAD

**Discussion by stillhope** [subdavis/kobo-book-downloader#107] [Q&A] [answered]
Hi, I would really like to dedrm a few KEPUB that I have discovered in my library sort out.  For each title, I have my **userkey** and **deviceID** of that time.  I do not use Calibre. Not installed.

I have python 2 and python 3 installed. I have a catalina and a ventura macbook. Not sure which one is easiest or if it makes a difference.

Is it possible? 
If so, what is involved ? I dont know python but maybe I can find someone who will help me (but they wont know this script).

thankyou kindly

**Comment by subdavis** [comment, marked as answer]
Converted this to a discussion issue.

You could do this using a Python interpreter.  Basically here are the steps:

```shell
python
from kobodl.koboDrmRemover import KoboDrmRemover
drmRemover = KoboDrmRemover("deviceId", "userId")
drmRemover.RemoveDrm(temporaryOutputPath, outputPath, contentKeys)
```

You can get your userId and deviceId from the `kobodl user add` process.

**Reply by stillhope** [reply]
Superb ! Thankyou ever so very much. I will try this.
